eras/babbage/impl/cardano-ledger-babbage.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-babbage
-version:            1.5.1.1
+version:            1.6.0.0
```

Comment:
Changelog needs to be updated with the new version and the breaking change

Reply:
done